user_data/strategies/hlhb.py

```diff
@@ -0,0 +1,128 @@
+# pragma pylint: disable=missing-docstring, invalid-name, pointless-string-statement
+
+import numpy as np  # noqa
+import pandas as pd  # noqa
+from pandas import DataFrame
+from freqtrade.strategy import IStrategy
+import talib.abstract as ta
+import freqtrade.vendor.qtpylib.indicators as qtpylib
+
+
+class hlhb(IStrategy):
+    """
+    The HLHB ("Huck loves her bucks!") System simply aims to catch short-term forex trends.
+    More information in https://www.babypips.com/trading/forex-hlhb-system-explained
+    """
+
+    INTERFACE_VERSION = 2
+
+    position_stacking = "True",
+
+    # Minimal ROI designed for the strategy.
+    # This attribute will be overridden if the config file contains "minimal_roi".
+    minimal_roi = {
+        "0": 0.6225,
+        "703": 0.2187,
+        "2849": 0.0363,
+        "5520": 0
+    },
+
+    # Optimal stoploss designed for the strategy.
+    # This attribute will be overridden if the config file contains "stoploss".
+    stoploss = -0.3211,
+
+    # Trailing stoploss
+    trailing_stop = "True",
+    trailing_stop_positive = 0.0117,
+    trailing_stop_positive_offset = 0.0186,
+    trailing_only_offset_is_reached = "True",
```

Comment:
The trailing stop (syntax) you use here is not valid - they'll end up as tuples (because of the trailing comma) - which will cause problems and errors all over the place.

```suggestion
    trailing_stop = True
    trailing_stop_positive = 0.0117
    trailing_stop_positive_offset = 0.0186
    trailing_only_offset_is_reached = True
```